packages/plugins/contracts/test/script/Cases.s.sol

```diff
@@ -40,62 +37,15 @@ contract Simple is Script, Sphinx {
         fallbackContract.set(1);
 
         // Deploys contract that deploys another contract in its constructor using create2
-        // The contract deployed is labeled
         ConstructorDeploysContract constructorDeploysContract = new ConstructorDeploysContract{
             salt: bytes32(uint(1))
         }(1);
-        sphinxLabel(
-            address(constructorDeploysContract.myContract()),
-            "contracts/test/ConstructorDeploysContract.sol:DeployedInConstructor"
-        );
 
         // Deploys contract that deploys another contract in its constructor using create3
-        // Both the parent and child are labeled
         bytes memory constructorDeploysContractInitCode = abi.encodePacked(
             type(ConstructorDeploysContract).creationCode,
             abi.encode(2)
         );
-        ConstructorDeploysContract constructorDeploysContractCreate3 = ConstructorDeploysContract(
-            CREATE3.deploy(bytes32(uint(1)), constructorDeploysContractInitCode, 0)
-        );
-        sphinxLabel(
-            address(constructorDeploysContractCreate3),
-            "contracts/test/ConstructorDeploysContract.sol:ConstructorDeploysContract"
-        );
-        sphinxLabel(
-            address(constructorDeploysContractCreate3.myContract()),
-            "contracts/test/ConstructorDeploysContract.sol:DeployedInConstructor"
-        );
-
-        // Deploys contract that deploys another contract in its constructor using create2
-        // The contract is not labeled
-        new ConstructorDeploysContract{ salt: bytes32(uint(2)) }(3);
-
-        // Deploys contract that deploys another contract in its constructor using create3
-        // Neither contract is labeled
-        bytes memory constructorDeploysContractUnlabeledInitCode = abi.encodePacked(
-            type(ConstructorDeploysContract).creationCode,
-            abi.encode(4)
-        );
-        CREATE3.deploy(bytes32(uint(2)), constructorDeploysContractUnlabeledInitCode, 0);
-
-        // Deploy a contract whose name is not unique in the source directory
-        // The contract is labeled
-        ConflictingNameContract conflictingNameContract = new ConflictingNameContract{ salt: 0 }(1);
-        sphinxLabel(
-            address(conflictingNameContract),
-            "contracts/test/conflictingNameContracts/First.sol:ConflictingNameContract"
-        );
-
-        // Deploy a contract whose name is not unique in the source directory
-        // The contract is not labeled
-        new ConflictingNameContract{ salt: bytes32(uint(1)) }(2);
-
-        // Deploy a contract whose name is not unique in the source directory
-        // We interact with the contract, so it does not require a label
-        ConflictingNameContract conflictingNameContractInteract = new ConflictingNameContract{
-            salt: bytes32(uint(2))
-        }(3);
-        conflictingNameContractInteract.set(5);
```

Comment:
Copying and pasting my comment from earlier:

> The `ConflictingNameContract` files aren't necessary anymore. For context, their purpose was to check that we can use labels for contract names that aren't unique in the repo, which was important to test because Foundry doesn't provide a fully qualified name in this scenario. These contracts, and the corresponding tests in `deploy.spec.ts`, aren't necessary anymore because we infer artifacts based on init code and not by fully qualified name.